adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -294,13 +297,16 @@ private AuthenticationResult acquireTokenSilentFlow(final AuthenticationRequest
     }
 
     /**
-     * Try acquire token silent locally.
-     */
+    * Try acquire token silent locally.
+    */
     private AuthenticationResult tryAcquireTokenSilentLocally(final AuthenticationRequest authenticationRequest)
             throws AuthenticationException {
         Logger.v(TAG, "Try to silently get token from local cache.");
         final AcquireTokenSilentHandler acquireTokenSilentHandler = new AcquireTokenSilentHandler(mContext,
                 authenticationRequest, mTokenCacheAccessor);
+        if(mAuthContext.getExtendedLifetimeEnabled()) {
+            acquireTokenSilentHandler.setOutageModeIsOn(true);
+        }
```

Comment:
new line after }
nit: I prefer the have the code block pretty clear, and it's easy for people to read.